src/inference/include/openvino/runtime/intel_cpu/properties.hpp

```diff
@@ -63,5 +63,25 @@ static constexpr Property<bool> denormals_optimization{"CPU_DENORMALS_OPTIMIZATI
  */
 static constexpr Property<float> sparse_weights_decompression_rate{"CPU_SPARSE_WEIGHTS_DECOMPRESSION_RATE"};
 
+/**
+ * @brief This property defines group size for dynamic quantization optimization
+ * @ingroup ov_runtime_cpu_prop_cpp_api
+ *
+ * Dynamic quantization optimization provides an ability to get performance benefit from int8 compute.
+ * In contrast with static quantization dynamic approach assumes activations are quantized during inference.
+ * Despite the fact dynamic quantization has some runtime overheads, it might provide better accuracy metrics.
+ * This property defines granularity (aka block size) for dynamic quantization algorithms. Lower group size values
+ * might result in better accuracy, but the drawback is worse performance. Group size equal 0 means dynamic
+ * quantization optimization is disabled.
+ * CPU Plugin uses this property as hint and applies dynamic quantization optimization only for operations which
+ * can benefit from performance perspective (see user documentation for full list of limitations).
+ * The following code allows to set the dynamic quantization group size.
+ *
+ * @code
+ * core.set_property(ov::intel_cpu::dynamic_quantization_group_size(64));
+ * @endcode
+ */
+static constexpr Property<size_t> dynamic_quantization_group_size{"CPU_DYNAMIC_QUANTIZATION_GROUP_SIZE"};
```

Comment:
maybe we can explicitly use 32/64 bits to avoid discrepancies between OS versions?

Reply:
Changed to unint64_t